package.json

```diff
@@ -18,7 +18,7 @@
     "generateFiles": "node ./generator/generateFiles.js",
     "generateArray": "node ./generator/generateArray.js",
     "lint": "standard",
-    "code-fix": "standard --fix",
+    "code-fix": "standard --fix || exit 0",
```

Comment:
Let's add another version of this script. because as we have a lot of interns - we need to have warnings/reports

Reply:
Will be done in upcoming pull request